src/components/editCqlLibrary/EditCqlLibrary.tsx

```diff
@@ -457,6 +463,35 @@ const EditCqlLibrary = () => {
                     label="Experimental"
                   />
                 </div>
+
+                <div className="form-row">
+                  <AutoComplete
+                    id="programUseContext"
+                    name="programUseContext"
+                    dataTestId="programUseContext"
+                    label="Program Use Context"
+                    placeholder="-"
+                    disabled={!formik.values.draft || !isOwner}
+                    error={
```

Comment:
was wondering what potential errors could be there for this

Reply:
Since it isn't required and it's only a strict select there shouldn't really be any errors at this time...there might be in the future when we allow them to free-form enter other Program Use Context options that aren't in the list.
I could remove the error attribute here for now?

Reply:
fine with leaving it there.